test/core/transport/chaotic_good/BUILD

```diff
@@ -236,10 +235,14 @@ grpc_cc_test(
     ],
 )
 
-grpc_yodel_simple_test(
-    name = "control_endpoint",
+grpc_fuzz_test(
+    name = "control_endpoint_test",
     srcs = ["control_endpoint_test.cc"],
-    external_deps = ["gtest"],
+    external_deps = [
```

Comment:
Nit: if these are common among for all grpc_fuzz_test definitions, can we always append these 3 dependencies inside the grpc_fuzz_test implementation (i.e., in the .bzl file) ?